libs/damap/src/lib/components/dmp/live-preview/live-preview.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="dialog-header">
+  <h1 mat-dialog-title>{{ "preview.title" | translate }}</h1>
+  <button mat-button mat-dialog-close class="top-right-button">
+    <mat-icon style="margin: 0">close</mat-icon>
+  </button>
+</div>
+
+<mat-dialog-content style="height: 100%; max-height: 90%">
+  <div class="container" style="height: 100%">
```

Comment:
shall we move all inline styling to the respective css files and just use an appropriate class here?